samples/application_development/sysbuild/with_mcuboot/README.rst

```diff
@@ -0,0 +1,52 @@
+.. _with_mcuboot:
+
+Sample with MCUboot
+###################
+
+Overview
+********
+A simple example that demonstrates how building a sample using sysbuild can
+automatically include MCUboot as bootloader.
+It show case how the sample can adjust the configuration of extra image by
+creating a image specific Kconfig fragment.
+
+Sysbuild specific settings
+**************************
+
+This sample automatically includes MCUboot as bootloader when built using
+sysbuild.
+
+This is achieved with a sysbuild specifc Kconfig configuration,
+:file:`sysbuild.conf`.
+
+The `SB_CONFIG_BOOTLOADER_MCUBOOT=y` setting in the sysbuild Kconfig file
+enables the bootloader when building with sysbuild.
+
+The :file:`sysbuild/mcuboot.conf` file will be used as an extra fragment that
+is merged together with the default configuration files used by MCUboot.
+
+:file:`sysbuild/mcuboot.conf` adjust the log level in MCUboot, as well as
+configure MCUboot to prevent downgrades.
```

Comment:
```suggestion
:file:`sysbuild/mcuboot.conf` adjusts the log level in MCUboot, as well as
configures MCUboot to prevent downgrades and operate in upgrade-only mode.
```

Reply:
applied, thanks :+1: